source/_integrations/roon.markdown

```diff
@@ -15,19 +15,14 @@ The Roon integration allows you to control [RoonLabs](https://roonlabs.com/) mus
 
 This integration uses Roon Core, a Roon application that runs on a machine on your network. Via Roon Core, Home Assistant can control all the Roon music players on your network.
 
-To integrate with Roon, you need to provide Home Assistant with the Hostname or IP address of the machine that runs your Roon Core, and then authorize Home Assistant in the Roon application.
-
-If you use an IP address, please assign a static IP address to the machine that runs Roon Core. This ensures that it won't change IP addresses, so you won't have to change the configuration in Home Assistant if it reboots and changes IP address. See your router's manual for details on how to set this up.
-
 ## Configuration
 
-You need the Hostname or IP address of the machine that runs your Roon Core. This might be a machine name (which can be followed by `.local`, e.g., `myserver.local`) or can be an IP address.
-
 1. From the Home Assistant front-end, navigate to **Configuration** then **Integrations**. Under **Set up a new integration** locate 'Roon' and click **Configure**.
-2. Enter the `Hostname` or `IP address` for the Roon Core machine and click **Submit**.
-3. Home Assistant will then contact your Roon Core and ask to be authorized. You will need to enable this extension in the Room Application. Go to **Settings** and then **Extensions**, there you will see an entry for Home Assistant with a button next to it. Click **Enable**.
-4. Roon core will then provide Home Assistant with the details of your media players.
-5. In Home Assistant you can then pick an area for each of your music players, and add them to Home Assistant.
+2. Home Assistant will then try find your Roon Core - if it is successful it will display `Authorize HomeAssistant in Roon`. Cllick **Submit** and skip to step 4
```

Comment:
```suggestion
2. Home Assistant will then try to find your Roon Core - if it is successful it will display `Authorize HomeAssistant in Roon`. Click **Submit** and skip to step 4
```